CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 ## [Unreleased]
 
 ### Changed
+- Implemented [#825](https://github.com/JabRef/jabref/issues/825): Search Bar across all bib files instead each having its own
+- Implemented [#573](https://github.com/JabRef/jabref/issues/573): Add key shortcut for global search (`ctrl+shift+f`, if the searchfield is empty it will be focused instead)
+- The search result Window will now show which entry belongs to which bib file
+- The search result Window will now remember its location
+- The search result Window won't stay on top anymore if the main Window is focused and will be present in the taskbar
+- The User can jump from the searchbar to the maintabel with `ctrl + enter`
```

Comment:
Keyboard shortcuts should be rendered consistenly without space before and after `+`